modules/ardrivo/src/Servo.cxx

```diff
@@ -0,0 +1,78 @@
+/*
+ *  Servo.cxx
+ *  Copyright 2020 ItJustWorksTM
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+#include "Arduino.h"
+#include "BoardDataDef.hxx"
+#include "Servo.h"
+
+constexpr int servo_min(int8_t min) { return MIN_PULSE_WIDTH - min * 4; }
+constexpr int servo_max(int8_t max) { return MAX_PULSE_WIDTH - max * 4; }
+constexpr uint8_t TRIM_DURATION = 2;
+
+Servo::Servo() {
+    is_attached = false;
```

Comment:
Either set it false by default in the header file (`bool is_attached = false;`) 
or do `Servo::Servo() : is_attached{false} {}`

Reply:
Any specific reason?

Reply:
Just convention mostly (And to be sure the value is initialized), since the only thing you're doing is setting the value there's no need to put it in the constructor body.

Reply:
ah, okay. what should we do in the constructor? Since there is nothing need to be initialized in this class before the pin is attached.

Reply:
If the constructor is completely empty just remove it :) or in the header `Servo() = default;`

Reply:
`Servo() = default` sound good